src/screens/signup/otpVerification/style.ts

```diff
@@ -22,3 +31,62 @@ export const HeaderTitleLabel = styled.Text`
   text-transform: capitalize;
   padding: 0px 5px;
 `;
+
+export const HeaderTitleOf = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.MEDIUM_SIZE + 3}px;
+  font-family: ${({ theme }) => theme.fonts.CORMORANT_REGULAR};
+  color: ${({ theme }) => theme.colors.FONT_DARK_COLOR};
+  text-transform: capitalize;
+  padding: 0px 5px;
+`;
+
+export const HeaderTitleNumber = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.MEDIUM_SIZE + 3}px;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  color: ${({ theme }) => theme.colors.FONT_DARK_COLOR};
+  text-transform: capitalize;
+`;
+
+export const CancelSetupButton = styled.TouchableOpacity`
+  width: ${applyScale(25)}px;
+  height: ${applyScale(25)}px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 25px;
+  background-color: ${({ theme }) => theme.colors.BUTTON_DARK_COLOR};
+`;
+
+export const Text = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.MEDIUM_SIZE}px;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  text-align: center;
+  margin-left: 40%;
+`;
+
+export const ButtonStyle = {
+  mainButton: {
+    width: '100%'
+  },
+  textStyle: {
+    fontFamily: theme.fonts.JOST_MEDIUM,
+    fontSize: 15
+  }
+};
+
+export const Title = styled.Text`
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  width: 311px;
+  font-size: 30px;
+  text-align: left;
+  height: 43px;
```

Comment:
for all static heights use applyScale method on it to be fluid on all pages instead of fixed points as 43px might look some worth different on a smaller phone than that of a tab screen, applyScale works just like percentages